Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -137,9 +137,15 @@ static QualifiedName subtitleAttr()
     return QualifiedName { nullAtom(), "subtitle"_s, nullAtom() };
 }
 
+static const AtomString& saveEvent()
```

Comment:
You can name this `saveAtom()`, it would be less odd especially since you're using it in `saveAttr()`